src/main/python/transformation/sample_observation_period_query.py

```
@@ -0,0 +1,115 @@
+from __future__ import annotations
+
+from sqlalchemy import func, select, join, union, literal
+from sqlalchemy.sql.expression import Insert
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from src.main.python.wrapper import Wrapper
+
+
+def observation_period_query(wrapper: Wrapper) -> Insert:
+
+    person = wrapper.get_cdm_table('person')
+    condition = wrapper.get_cdm_table('condition_occurrence')
+    visit = wrapper.get_cdm_table('visit_occurrence')
+    drug = wrapper.get_cdm_table('drug_exposure')
+    procedure = wrapper.get_cdm_table('procedure_occurrence')
+    observation = wrapper.get_cdm_table('observation')
+    measurement = wrapper.get_cdm_table('measurement')
+    obs_period = wrapper.get_cdm_table('observation_period')
+    death = wrapper.get_cdm_table('death')
```

...However now the code relies on this returning a `None`. I think it would be better to throw an error, and convert the later `if..else` into a `try..except` clause that includes this assignment.